camp_2/buidl/protostar-buidl/tests/test_main.cairo

```diff
@@ -0,0 +1,28 @@
+%lang starknet
```

Comment:
is this an actual test or something that was left over from an older example?

Reply:
It is the test file that protostar creates by default, I will eliminate it. Thanks!